package.json

```diff
@@ -9,13 +9,28 @@
     "url": "git://github.com/TradePending/snap-sdk.git"
   },
   "dependencies": {
-    "jquery": "^3.5.0"
+    "jquery": "^3.7.0"
```

Comment:
There's no CVEs for jquery 3.5 or later.  You just doing this for funsies?  It might be a minor inconvenience to the potentially non-existent users of this package.